src/cljs/main/broadfcui/config.cljs

```diff
@@ -57,6 +57,7 @@
 (defn call-caching-guide-url [] (get @config "callCachingGuideUrl"))
 (defn google-bucket-url [filename] (str (get @config "googleBucketUrl") filename ".json"))
 (defn martha-url [] (get @config "marthaUrl"))
+(defn metrics-url [] (get @config "metricsUrl"))
```

Comment:
TOL: This will need a corresponding firecloud-develop PR, right?

Reply:
Yup, and that will also be what enables it.